packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js

```diff
@@ -1452,6 +1452,18 @@ const tests = {
         }
       `,
     },
+    {
```

Comment:
Do we have a way to gate eslint tests? Not sure if it's worth it to wrap the ExhaustiveDeps check with the experimental flag.

Reply:
Gonna remove it for now, not sure it's worth the effort.